src/views/v2/list/event/venue.php

```diff
@@ -20,7 +20,7 @@
 	return;
 }
 
-$separator            = esc_html_x( ', ', 'Address separator', 'events-calendar-pro' );
+$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar-pro' );
```

Comment:
```suggestion
$separator            = esc_html_x( ', ', 'Address separator', 'tribe-events-calendar' );
```

Reply:
@Camwyn then it should be 'the-events-calendar' !